audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/kafka/KafkaContractTest.java

```diff
@@ -104,33 +114,30 @@
 @Import(KafkaContractTest.KafkaInitializer.Config.class)
 @ContextConfiguration(initializers = {KafkaContractTest.KafkaInitializer.class})
 @ActiveProfiles({"akka-test"})
+@ExtendWith(MockitoExtension.class)
```

Comment:
Does not look like we need @ExtendWith to run these tests.

Reply:
Yup, just tested it without, so I 've removed it.